diff --git a/node_modules/next-auth/dist/server/routes/session.js b/node_modules/next-auth/dist/server/routes/session.js
index 73e5285..10bd998 100644
--- a/node_modules/next-auth/dist/server/routes/session.js
+++ b/node_modules/next-auth/dist/server/routes/session.js
@@ -102,7 +102,8 @@ var _default = function () {
             user: {
               name: user.name,
               email: user.email,
-              image: user.image
+              image: user.image,
+              id: user.id
             },
             accessToken: session.accessToken,
             expires: session.expires
